tests/package.json

```diff
@@ -3,7 +3,8 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "@kiltprotocol/sdk-js": "^0.19.0",
+    "@kiltprotocol/core": "^0.21.0",
```

Comment:
Should either be one or the other. Since `sdk.js` has core features. 

I would suggest for the tutorial remove the `core` and we could write a line of closing text like... Add individual packages or the whole thing with `sdk-js`.